### PR TITLE
New playbooks and incident type for PAN-OS Policy Optimizer

### PR DESCRIPTION
**Pull request by mayagoldb**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://github.com/demisto/etc/issues/35563

## Description
New playbooks and incident type for PAN-OS Policy Optimizer. Playbooks include:
- Policy Optimizer - Manage Unused Rules
- Policy Optimizer - Manage Rules with Unused Applications
- Policy Optimizer - Manage Port Based Rules

## Minimum version of Cortex XSOAR
- [x] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [ ] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/PANOSPolicyOptimizer/Scripts/EntryWidgetPortBasedRules/EntryWidgetPortBasedRules.py [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+                        "value": 3
+                    }
+                }
+            },
+            "type": "above"
+        }
+    }
+}
+
+demisto.results(data)
```
Comment (yaakovi) [inline Packs/PANOSPolicyOptimizer/Scripts/EntryWidgetPortBasedRules/EntryWidgetPortBasedRules.py:35]: in all scripts
```suggestion
return_results(data)
```

File: Packs/PANOSPolicyOptimizer/Scripts/EntryWidgetPortBasedRules/EntryWidgetPortBasedRules.py [hunk 2]
```diff
@@ -0,0 +1,35 @@
+import demistomock as demisto  # noqa: F401
+from CommonServerPython import *  # noqa: F401
+
+
+incident = demisto.incidents()
```
Comment (yaakovi) [inline Packs/PANOSPolicyOptimizer/Scripts/EntryWidgetPortBasedRules/EntryWidgetPortBasedRules.py:5]: this is not used, remove

**Review by yaakovi — COMMENTED** [review]
your lint is failing due to an error in the demistomock.
a fix will be available after merging [this pr](https://github.com/demisto/content/pull/16213).

only reviewed the scripts

**Comment by mayagoldb** [comment]
@yaron-libman new playbooks for the Policy Optimizer packs, please review the playbooks and doc files

**Comment by altmannyarden** [comment]
@julieschwartz18 related article waiting for doc review as well https://github.com/demisto/content-docs/pull/880

**Comment by julieschwartz18** [comment]
@mayagoldb - Another question for the Update playbook-Policy_Optimizer_-_Manage_Port_Based_Rules.yml - for line 1062 (in the updated file) - the description for the Ask by task only task is "Sends a message to a user, group, or channel." - this doesn't make sense (that makes sense as the description for the Send Slack message task)

**Comment by julieschwartz18** [comment]
@mayagoldb - I am not able to view a preview for the playbook-Policy_Optimizer_-_Manage_Rules_with_Unused_Applications.yml file (this is before any edits were applied)

**Comment by mayagoldb** [comment]
@julieschwartz18 the generic descriptions you mentioned are the integration tasks descriptions (I didn't adjust them specifically to this use-case), I will update

File: Packs/PANOSPolicyOptimizer/README.md [hunk 3]
```diff
@@ -1,7 +1,17 @@
...
-- Identifies port-based rules so you can convert them to application-based rules that allow traffic or adds applications to existing rules without compromising application availability. 
-- Identifies rules configured with unused applications. 
-- Helps you to analyze rule characteristics and prioritize which rules to migrate or clean up first.
+## What Does This Pack Do?
+The Policy Optimizer integration in this content pack enables you to gain visibility into and control usage of security policy rules.
+
+The playbooks in this pack also help you automate the following procedures to reduce the attack surface and safely enable applications on your network.
+
+- Identify port-based rules so you can convert them to application-based rules that allow traffic or add applications to existing rules without compromising application availability.
+- Identify rules configured with unused applications.
```
Comment (altmannyarden) [inline Packs/PANOSPolicyOptimizer/README.md:10]: Dont we have the unused rules, unused applications and port base?

File: Packs/PANOSPolicyOptimizer/ReleaseNotes/1_0_3.md [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+- **PAN-OS Port Based Rules**
+- **PAN-OS Rules with Unused Applications**
+- **PAN-OS Unused Rules**
+- **Policy Optimizer Description**
+- **Policy Optimizer Use-Case**
+
+#### Integrations
+Added the new **PAN-OS Policy Optimizer** integration.
+
+Updated the Docker image to: *XXXXX*.
```
Comment (altmannyarden) [inline Packs/PANOSPolicyOptimizer/ReleaseNotes/1_0_3.md:32]: what is that line? :O

File: Packs/PANOSPolicyOptimizer/Scripts/EntryWidgetPortBasedRules/EntryWidgetPortBasedRules.yml [hunk 5]
```diff
@@ -0,0 +1,19 @@
...
+name: EntryWidgetPortBasedRules
+runas: DBotWeakRole
+runonce: false
+script: ''
+scripttarget: 0
+subtype: python3
+tags:
+- dynamic-section
+type: python
+fromversion: 5.5.0
```
Comment (altmannyarden) [inline Packs/PANOSPolicyOptimizer/Scripts/EntryWidgetPortBasedRules/EntryWidgetPortBasedRules.yml:17]: If the pack / layout only from version 6 - we should change the fromversion of all the widgets to be fromversion 6

**Review by altmannyarden — APPROVED** [review]

**Pull request merged**